kiwi/kiwi_compat.py

```diff
@@ -189,22 +187,21 @@ def __repo_argument(self, source, repo_type, alias, priority):
 class Command(object):
     @classmethod
     def execute(self, arguments):
-        # legacy kiwi binary lives in /usr/sbin, the new kiwi
-        # lives in /usr/bin. In order to avoid conflicts the path
-        # to the kiwi we want to use is specificly set here
-        kiwi = '/usr/bin/kiwi'
+        kiwi = '/usr/bin/kiwi-ng'
```

Comment:
yes this will be a problem, we could change it to a relative lookup, that would require the path to be in the search path of the shell, if not bad luck but I could live with that. Thoughts ?